packages/api-clients/open-api/bffApi.yml

```
@@ -3356,6 +3356,115 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  /templates/eservices/{eServiceId}/descriptors/{descriptorId}/update:
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+    post:
+      security:
+        - bearerAuth: []
+      tags:
+        - eservices
+      summary: Update the selected instance descriptor.
```

Align with other template routes

```suggestion
      summary: Update the selected template instance descriptor.
```